docker-image/run-nlp-profiler-in-docker.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Try to get familiar with a bash shell linter to see what comments it has about your shell script - search for `shellcheck`, install it and then use it on this shell script.

Use the comments from `shellcheck` to improve the code if any comments are given